content/en/blog/_posts/2024-05-13-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,267 @@
+presented by clients connecting to the `foo-https` Gateway Listener.
+
+```
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: client-validation-basic
+spec:
+  gatewayClassName: acme-lb
+  listeners:
+    name: foo-https
+    protocol: HTTPS
+    port: 443
+    hostname: foo.example.com
+  tls:
+    certificateRefs:
+      kind: Secret
+      group: ""
+      name: foo-example-com-cert
+    frontendValidation:
+      caCertificateRefs:
+        kind: ConfigMap
+        group: ""
+        name: foo-example-com-ca-cert
+```
+
+Leading Contributor: Arko Dasgupta
+
+#### [Session Persistence and BackendLBPolicy](https://gateway-api.sigs.k8s.io/geps/gep-1619/)
+
+[Session Persistence](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io%2fv1.SessionPersistence) is being introduced to Gateway API via a new policy
+([BackendLBPolicy](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1alpha2.BackendLBPolicy))
+for Service-level configuration and as fields within HTTPRoute
+and GRPCRoute for route-level configuration. The BackendLBPolicy and route-level
+APIs provide the same session persistence configuration, including session
+timeouts, session name, session type, and cookie lifetime type.
+
+Below is an example configuration of `BackendLBPolicy` that enables cookie-based
+session persistence for the `foo` service. It sets the session name to
+`foo-session`, defines absolute and idle timeouts, and configures the cookie to
+be a session cookie:
+
+```
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: BackendLBPolicy
+metadata:
+  name: lb-policy
+  namespace: foo-ns
+spec:
+  targetRefs:
+  - group: core
+    kind: service
+    name: foo
+  sessionPersistence:
+    sessionName: foo-session
+    absoluteTimeout: 1h
+    idleTimeout: 30m
+    type: Cookie
+    cookieConfig:
+      lifetimeType: Session
+```
+
+Leading Contributors: Grant Spence, Gina Yeh
+
+#### [TLS Terminology Clarifications](https://gateway-api.sigs.k8s.io/geps/gep-2907/)
+
+As part of a broader goal of making our TLS terminology more consistent
+throughout the API, we've introduced some breaking changes to BackendTLSPolicy.
+This has resulted in a new API version (v1alpha3) and will require any existing
+users of this policy to uninstall the v1alpha2 version before installing this
+newer version.
+
+Any references to v1alpha2 BackendTLSPolicy fields will need to be updated to
+v1alpha3. Specific changes to fields include:
+- `targetRef` becomes `targetRefs` to allow a BackendTLSPolicy to attach to
+multiple targets
+- `tls` becomes `validation`
+- The type name BackendTLSPolicyConfig becomes BackendTLSPolicyValidation
+- `tls.caCertRefs` becomes `validation.caCertificateRefs`
+- `tls.wellKnownCACerts` becomes `validation.wellKnownCACertificates`
+- The type name WellKnownCACertType becomes WellKnownCACertificatesType
+and
+- The constant WellKnownCACertSystem becomes WellKnownCACertificatesSystem
```

Comment:
I'd recommend leaving out any reference to go types since our primary audience for this blog will be people using the APIs directly with something like YAML, not Go types.

Reply:
I could totally get behind this, FTR. 👍

Reply:
Done.